docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -147,7 +147,7 @@ The content of the *Source* tab automatically updates when you edit values on th
 . Click *Save*.
 
 The new design is listed on the *API and Schema Designs* page.
-You can use the options icon (three vertical dots) to view details about the design (including export and import events, and the edit history), rename the design, open the design editor, download the design to a file, export the design to {product-long-registry}, or delete the design.
+You can use the options icon (three vertical dots) to view details about the design (including export and import events, and the edit history), edit the design metadata (such as the name and description), open the design editor to edit the design content, export the design to {product-long-registry}, download the design to a file, or delete the design.
```

Comment:
This is quite a long sentence with a long list, and with a lot of parentheses. Suggest one of following options:

-  Cut it back to something like: 

"You can use the options icon (three vertical dots) to view or edit the details about the design, edit the design content, export the design to {product-long-registry}, download the design to a file, or delete the design."

- Add a bulleted list something like:  

"You can use the options icon (three vertical dots) to perform the following tasks:
 -- View details about the design, such as export or import events, or edit history 
 -- Edit the design metadata, such as name and description 
 -- Edit the design content 
 -- Export the design to {product-long-registry} 
 -- Download the design to a file
 -- Delete the design

See also https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses

Reply:
My preference would be the first one as a summary, the option smenu should speak for itself, so no need for the doc to list out all available options?

Reply:
Done!